spec/models/elastic_sayt_suggestion_spec.rb

```diff
@@ -38,33 +56,32 @@
           end
 
           it 'should return zero results' do
-            search = ElasticSaytSuggestion.search_for(q: 'suggests', affiliate_id: affiliate.id, size: 1, offset: 1, language: affiliate.indexing_locale)
+            search = ElasticSaytSuggestion.search_for(search_params)
             expect(search.total).to be_zero
             expect(search.results.size).to be_zero
           end
         end
       end
-
     end
   end
 
-  pending "highlighting results" do
+  describe 'highlighting results' do
     before do
       affiliate.sayt_suggestions.create!(phrase: 'hi suggest me', popularity: 30)
       ElasticSaytSuggestion.commit
     end
 
     context 'when no highlight param is sent in' do
       it 'should highlight appropriate fields with default highlighting' do
-        search = ElasticSaytSuggestion.search_for(q: 'suggests', affiliate_id: affiliate.id, language: affiliate.indexing_locale)
+        search = ElasticSaytSuggestion.search_for(search_params)
         first = search.results.first
         expect(first.phrase).to eq("hi <strong>suggest</strong> me")
       end
     end
 
     context 'when highlight is turned off' do
       it 'should not highlight matches' do
-        search = ElasticSaytSuggestion.search_for(q: 'suggests', affiliate_id: affiliate.id, language: affiliate.indexing_locale, highlighting: false)
+        search = ElasticSaytSuggestion.search_for(q: 'suggest', affiliate_id: affiliate.id, language: affiliate.indexing_locale, highlighting: false)
```

Comment:
This line is very long. You can break it something like below. Or even better I would make it its own variable as rails 5 we have to also assign it to params hash. 

```
search = ElasticSaytSuggestion.search_for(
  q: 'suggest', 
  affiliate_id: affiliate.id, 
  language: affiliate.indexing_locale, 
  highlighting: false
)
```